### ISSUE TITLE
Assertion package

### ISSUE THREAD

**Issue by queso** [xdissent/meteor-cucumber#5]
Have you put any thought into including something like Chai for assertion support?

**Comment by xdissent** [comment]
I envision additional meteorite packages that extend/add cucumber World classes with support for various other libraries. The protractor, zombie, and selenium worlds would probably move to separate packages at that point too. Open to suggestion though

**Comment by queso** [comment]
I guess I don't fully grok the cucumber world classes much, so I am not sure how I would hook that in.

Assuming we had a Chai world, how would that reconcile with a Zombie world?

**Comment by queso** [comment]
This actually turned out easier than I thought.  I added chai `mrt add chai` and then just put `chai.should()` in my world constructor and now all my tests have the should syntax of chai enabled!

**Comment by xdissent** [comment]
Awesome! I was thinking about it last night and things like assertion libraries probably don't need an actual world, since they're usually loaded globally. Glad it worked out!

**Comment by queso** [comment]
Yeah, this seems easier enough to do, just need to doc it in the readme.
